src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -80,6 +80,16 @@ export class HaAutomationEditor extends LitElement {
 
   @internalProperty() private _entityId?: string;
 
+  public connectedCallback() {
+    super.connectedCallback();
+    window.addEventListener("keydown", this._keydownEvent);
```

Comment:
Can we add the listener to this element instead of window?

Reply:
Done.